modules/thermal_hydraulics/test/tests/components/hs_coupler_2d2d_radiation/energy_conservation.i

```diff
@@ -0,0 +1,202 @@
+# This input file is used to test that HSCoupler2D2DRadiation conserves
+# energy for a problem where 3 cylindrical heat structures (surfaces 1, 2, and 3)
+# are enclosed by an annular heat structure (surface 4).
```

Comment:
This is what is modeled by the HSCoupler2D2DRadiation, but the physical location of the heat structures doesn't match this description.